docs/Resources/resources.md

```diff
@@ -70,12 +75,11 @@ Source code for [the web version](https://github.com/openworm/wormbrowser) and [
 Data sets
 ---------
 
-OpenWorm Database
------------------
+### PyOpenWorm
 
-An web version of the OpenWorm database can [be browsed online](http://www.interintelligence.org/openworm/Entities.aspx).
-
-More information about working with the data within it and other data entities can be found on [the data representation project page](/Projects/datarep/).
+A straightforward Python API for accessing information about *C. elegans* is available
+at [PyOpenWorm](http://github.com/openworm/PyOpenWorm).  This API consolidates
+information from our various [publicly accessible archive of data sets](https://drive.google.com/#folders/0B_t3mQaA-HaMejlrMmpnR2VjN0U) that we have come across and adapted on Google Drive.
```

Comment:
Is this change ok?

Reply:
PyOpenWorm does not exist ;-)
It was renamed to owmeta. I would remove the "straightforward" part since it's redundant: we don't set out to make baroque/complicated APIs.

Reply:
again, not sure exactly what you want here. Would be extremely prefereble if you could just commit to this branch :)